pkg/controller/helper.go

```diff
@@ -91,20 +92,49 @@ func (o IssuerOptions) CanUseAmbientCredentials(iss cmapi.GenericIssuer) bool {
 	return false
 }
 
-func (o IssuerOptions) CertificateNeedsRenew(cert *x509.Certificate, renewBefore *metav1.Duration) bool {
-	renewBeforeDuration := o.RenewBeforeExpiryDuration
```

Comment:
IssuerOptions is no longer checked, meaning that we will ignore the option

Reply:
Updated - the CLI flag now defaults to `cmapi.DefaultRenewBefore` and we now only check `o.RenewBeforeExpiryDuration`